.github/workflows/main.yml

```diff
@@ -165,25 +165,21 @@ jobs:
 
   build-only-sysroot:
     name: Build only sysroot
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
       - uses: ./.github/actions/checkout
       - uses: ./.github/actions/install-deps
-      - run: |
-          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-          name=$(lsb_release -s -c)
-          sudo add-apt-repository -y "deb http://apt.llvm.org/$name/ llvm-toolchain-$name-18 main"
-          sudo add-apt-repository -y "deb-src http://apt.llvm.org/$name/ llvm-toolchain-$name-18 main"
-          sudo apt-get install -y clang-18 llvm-18 lld-18
-      - run: cargo install wasm-component-ld@0.5.5
+      - run: cargo install wasm-component-ld@0.5.12
       - run: |
           cmake -G Ninja -B build -S . \
             -DCMAKE_C_COMPILER=/usr/lib/llvm-18/bin/clang \
             -DCMAKE_SYSTEM_NAME=WASI \
-            -DWASI_SDK_INCLUDE_TESTS=ON
+            -DWASI_SDK_INCLUDE_TESTS=ON \
+            -DCMAKE_C_LINKER_DEPFILE_SUPPORTED=OFF \
+            -DCMAKE_CXX_LINKER_DEPFILE_SUPPORTED=OFF
```

Comment:
This is need here _and_ below?

Reply:
Unfortunately I believe so yeah. This layer is for the cmake build of wasi-sdk itself, aka when building tests. The cmake layer is then for the sub-cmake-builds of libraries like libcxx themselves. At least from CI it looked like both were necessary to get things passing when just using "clang" as the compiler.